src/main/java/com/spartronics4915/frc2019/Constants.java

```diff
@@ -63,10 +63,10 @@
     // Units: setpoint, error, and output are in ticks per second.
     public static final int kPositionPIDSlot = 0; // for compat with 2018
     public static final int kVelocityPIDSlot = 1; // for compat with 2018
-    public static final double kDriveVelocityKp = 1;
+    public static final double kDriveVelocityKp = 1.0;
     public static final double kDriveVelocityKi = 0.0;
     public static final double kDriveVelocityKd = 0.0;
-    public static final double kDriveVelocityKf = 0.5;
+    public static final double kDriveVelocityKf = 0.0;
```

Comment:
hm - we know this needs to be a non-zero value.  perhaps a comment here on units and why its currently set to 0 would be helpful.

Reply:
Why does it need to be non-zero? Don't we actually *want* it to be zero, because we're going to generate feedforward in another place?

Reply:
in a drivetrain, the whole idea that a single Kf is sufficient is bogus, so perhaps we just want to get rid of the constant altogether and route all computations of Kf through a single method call, one that indirects through our drive characterization constants.